payload.h

```diff
@@ -134,8 +134,8 @@
 
 #define PAYLOAD_FILENAME "nmap-payloads"
 
-const char *get_udp_payload(u16 dport, size_t *length);
-const char *udp_port2payload(u16 dport, size_t *length);
```

Comment:
Thoughts on updating `udp_port2payload` so that it is still declared in this header file?